<error errorId="193b91a8-ed7d-4464-9432-c0e4d6a2167f" host="ASUS" type="ElmahEFLogger.CustomElmahLogger.ElmahEfInterceptorException" message="Invalid operation. The connection is closed." detail="System.Exception: &#xD;&#xA;Command: &#xD;&#xA;INSERT [dbo].[Roles]([Description], [IsSystemRole], [IsDefaultForRegister], [IsActive], [Name])&#xD;&#xA;VALUES (@0, @1, @2, @3, @4)&#xD;&#xA;SELECT [Id]&#xD;&#xA;FROM [dbo].[Roles]&#xD;&#xA;WHERE @@ROWCOUNT &gt; 0 AND [Id] = scope_identity()&#xD;&#xA;&#xD;&#xA;Parameters: &#xD;&#xA;-- @0: 'مدیران' (Type = String, Size = 1024)&#xD;&#xA;-- @1: 'True' (Type = Boolean)&#xD;&#xA;-- @2: 'False' (Type = Boolean)&#xD;&#xA;-- @3: 'True' (Type = Boolean)&#xD;&#xA;-- @4: 'Administrators' (Type = String, Size = 256)&#xD;&#xA;&#xD;&#xA;OriginalException:&#xD;&#xA;System.InvalidOperationException: Invalid operation. The connection is closed.&#xD;&#xA;   at System.Data.SqlClient.SqlCommand.&lt;ExecuteDbDataReaderAsync&gt;b__24(Task`1 result)&#xD;&#xA;   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()&#xD;&#xA;   at System.Threading.Tasks.Task.Execute() &#xD;&#xA; ---&gt; ElmahEFLogger.CustomElmahLogger.ElmahEfInterceptorException: Invalid operation. The connection is closed.&#xD;&#xA;   --- End of inner exception stack trace ---" time="2015-07-20T06:29:33.9327945Z" />